cirq/ops/common_gates.py

```diff
@@ -1060,46 +1060,48 @@ class ISwapPowGate(eigen_gate.EigenGate,
     When exponent=1, swaps the two qubits and phases |01⟩ and |10⟩ by i. More
     generally, this gate's matrix is defined as follows:
 
-        ISWAP**t ≡ exp(+i π t (X⊗X + Y⊗Y) / 4)
+        ISWAP**t ≡ exp(-i π t (X⊗X + Y⊗Y) / 4)
 
     which is given by the matrix:
 
         [[1, 0, 0, 0],
-         [0, c, i·s, 0],
-         [0, i·s, c, 0],
+         [0, c, s, 0],
+         [0, s, c, 0],
          [0, 0, 0, 1]]
 
     where:
 
         c = cos(π·t/2)
-        s = sin(π·t/2)
+        s = -i·sin(π·t/2)
 
     `cirq.ISWAP`, the swap gate that applies i to the |01> and |10> states,
     is an instance of this gate at exponent=1.
```

Comment:
Is the sign of i wrong here now?  This was just changed from -1 a few days ago in #1848.

Reply:
I think we want this to be consistent with FSim.  #1882